File: coordinate_geometry.cpp
#include <iostream>
#include <cmath>   // sqrt için
using namespace std;

int main() {
    double x1, y1, x2, y2;

    cout << "Enter x1: ";
    cin >> x1;

    cout << "Enter y1: ";
    cin >> y1;

    cout << "Enter x2: ";
    cin >> x2;

    cout << "Enter y2: ";
    cin >> y2;

    double distance = sqrt((x2 - x1)*(x2 - x1) + (y2 - y1)*(y2 - y1));

    cout << "Distance is: " << distance << endl;

    return 0;
}

File: coordinate_geometry.py
import math  # python karekökü bilmiyormuş
x1=float(input("Enter X1:"))
y1=float(input("Enter Y1:"))
x2=float(input("Enter X2:"))
y2=float(input("Enter Y2:"))

distance = math.sqrt((x2-x1)**2 + (y2-y1)**2)
print(f"Distance: {distance} .")
